BoardUserInterface.API/Controllers/V1/TemplateController.cs

```diff
@@ -39,5 +42,28 @@ public async Task<IActionResult> RemoveAllVersions()
             var files = _templateService.RemoveAllVersionsAsync();
             return Ok("All template versions have been removed successfully.");
         }
+
+        [HttpGet("download")]
+        public IActionResult Download()
+        {
+            try
```

Comment:
The try catch is inside the service not in controller

Reply:
Made the changes